src/sentry/api/event_search.py

```diff
@@ -229,6 +232,7 @@ class SearchVisitor(NodeVisitor):
             # so they can be used in conditions
         ]
     )
+    boolean_keys = set(["error.handled"])
```

Comment:
If we do use this, I'm not sure what other keys would make sense to be in here.

Reply:
There are some device related fields like device.online, device.charging

Reply:
I added `stack.in_app` as well based on the name